android/build.gradle

```diff
@@ -65,7 +65,7 @@ android {
      * The following variables:
      * - androidBuildToolsVersion,
      * - androidCompileSdkVersion
-     * - qt5AndroidDir - holds the path to qt android files
+     * - qtAndroidDir - holds the path to qt android files
      *                   needed to build any Qt application
```

Comment:
fix the indentation here

Reply:
Docs, yes. :) 
rename, no. this property is set by the "qtDeployToAndroid" tool